e2e/startupSmoke.spec.ts

```diff
@@ -1,8 +1,72 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, Page, type Locator, BrowserContext } from '@playwright/test';
 
-test('has title', async ({ page }) => {
-    await page.goto('/');
+class DiveInfoPage {
+    private readonly titleSelector = '#dive-info-title';
+    private readonly timeLabelSelector = '#total-dive-time';
+    private readonly timeValueSelector = '#total-dive-time-value';
+    private readonly waypointTableSelector = '#dive-waypoints-table';
+    private readonly waypointRowsSelector = '#dive-waypoints-table tbody tr';
 
-    await expect(page).toHaveTitle(/Dugong Gas Planner/);
-});
+    constructor(private page: Page) {}
+
+    async navigate() {
+        await this.page.goto('http://localhost:4200/');
+    }
+
+    getTitle(): Locator {
+        return this.page.locator(this.titleSelector);
+    }
+
+    getTimeLabel(): Locator {
+        return this.page.locator(this.timeLabelSelector);
+    }
+
+    getTimeValue(): Locator {
+        return this.page.locator(this.timeValueSelector);
+    }
+
+    getWaypointTable(): Locator {
+        return this.page.locator(this.waypointTableSelector);
+    }
+
+    getWaypointRows(): Locator {
+        return this.page.locator(this.waypointRowsSelector);
+    }
+}
+
+test.describe('Dive Info Tests', () => {
+    let diveInfoPage: DiveInfoPage;
+    let context: BrowserContext;
+    let page: Page;
 
+    test.beforeEach(async ({ browser }) => {
+        context = await browser.newContext();
+
+        page = await context.newPage();
+
+        diveInfoPage = new DiveInfoPage(page);
+        await diveInfoPage.navigate();
+    });
+
+    test('should show dive info', async () => {
+        const title = diveInfoPage.getTitle();
```

Comment:
We dont need to check Title and Total dive time prefix, since they are static.
Checking for total dive time is enough.